util/CosmosHelper.js

```diff
@@ -48,94 +51,92 @@ export function amountToLIKE(likecoin) {
 }
 
 export async function getTransactionCompleted(txHash) {
-  if (!cosmosClient) await initCosmosClient();
-  const txData = await cosmosClient.getTx(txHash);
+  if (!stargateClient) await initStargateClient();
+  const txData = await stargateClient.getTx(txHash);
   if (!txData || !txData.height) {
     return 0;
   }
   const {
-    timestamp,
+    height,
     code,
-    rawLog,
   } = txData;
-  const fullLogs = JSON.parse(rawLog);
+  const block = await stargateClient.getBlock(height);
+  const { header: { time: timestamp } } = block;
+
   return {
     ts: (new Date(timestamp)).getTime(),
-    isFailed: (code && code !== '0') || !fullLogs[0].success,
+    isFailed: (code && code !== '0'),
```

Comment:
Is `code` a string or a number in the new cosmjs getTx() result?

Reply:
it's now a number. Will change it to !== 0